### ISSUE TITLE
Area Selection Zoom

### ISSUE THREAD

**Issue by banisadr** [foxglove/studio#4848]
Ability to select an area on a plot and have X/Y bounds zoom to the selected area

**Comment by TBdt38** [comment]
I agree it would be nice to have area zoom feature. Just being able to zoom on X axis is little bit limited.
